src/infrastructure/database/models.py

```diff
@@ -0,0 +1,129 @@
+from sqlalchemy import (
+    Column,
+    String,
+    DateTime,
+    Boolean,
+    JSON,
+    ForeignKey,
+    Integer,
+    Table,
+    func,
+)
+from sqlalchemy.orm import declarative_base, relationship
+from datetime import datetime
+from src.domain.notion.entities import NotionDocument, NotionUser
+import uuid
+
+Base = declarative_base()
+
+# Association tables for many-to-many relationships
+document_creators = Table(
+    "document_creators",
```

Comment:
let's prefix this table name with "notion_" also because we're going to be sharing the database with the discord bot